src/content/eol/2024/08/eol-08-01-24.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "EOL for Google Core Web Vital FID metric"
+summary: "On September 9, 2024, Google is deprecating the FID metric, in favor of Interaction to Next Paint (“INP”)."
+publishDate: '2024-08-01'
+eolEffectiveDate: '2024-09-09'
+---
+
+On September 9, 2024, [Google is deprecating the FID metric](https://developers.google.com/search/blog/2023/05/introducing-inp), in favor of Interaction to Next Paint (“INP”). Since New Relic uses certain libraries and APIs that leverage the FID metric, you will need to take the actions below before September 9, 2024 to avoid inaccurate or unexpected behavior with dashboards, alerts, and service level reporting. 
```

Comment:
Here and elsewhere in this doc, I suggest saying `custom dashboards` to avoid the impression that built-in/curated dashboards in the Browser UI will break.